CONTRIBUTORS.md

```diff
@@ -0,0 +1,2 @@
+Amirouche Amazigh BOUBEKKI <amirouche@hyper.dev>
+Lassi Kortela <lassi@lassi.io>
```

Comment:
I'd prefer to keep out my email address so spammers cannot harvest it with automated crawlers. If other people want to list their addresses, that's fine by me.

The addresses can be found from the Git history via `git shortlog -sne`.